Example/Tests/ContentMetadataTest.swift

```diff
@@ -114,22 +114,36 @@ class ContentMetadataTest: QuickSpec {
 
             context("when updating session") {
                 it("update video duration") {
+                    _ = TestDouble(aClass: playerDouble!, name: "duration", return: 50.0)
+
                     playerDouble.fakePlayEvent() // to initialize session
-                    var metadata = MetadataOverrides()
-                    metadata.duration = 50
-                    convivaAnalytics.updateContentMetadata(metadataOverrides: metadata)
                     let spy = Spy(aClass: CISVideoAnalyticsTestDouble.self, functionName: "setContentInfo")
 
                     expect(spy).to(
                         haveBeenCalled(withArgs: ["duration": "50"])
                     )
```

Comment:
This test was testing the `MetadataOverrides` feature instead of the default metadata collection feature.